package.json

```diff
@@ -21,14 +21,15 @@
     "body-parser": "^1.14.2",
     "express": "^4.13.4",
     "lodash": "^4.2.0",
+    "mongoose": "^4.4.3",
     "morgan": "^1.6.1",
+    "node-slack": "0.0.7",
     "nodemon": "^1.8.1",
     "nunjucks": "^2.3.0",
     "sass": "^0.5.0",
+    "standard": "^3.0.0",
     "webpack": "^1.12.12",
-    "winston": "^2.1.1",
-    "mongoose": "^4.4.3",
-    "standard": "^3.0.0"
+    "winston": "^2.1.1"
```

Comment:
Thank you for making this alphabetical! I'm going to test this PR heavily, but I am really excited.

One thing would be that we might not want so much spam in the #general channel. Some people might not enjoy that.

Perhaps set the channel to something like #server-logs, to keep it separate from the rest of the chatter :smile: